pilot/pkg/networking/core/route/route.go

```diff
@@ -1242,6 +1242,12 @@ func BuildDefaultHTTPInboundRoute(clusterName string, operation string) *route.R
 		// gRPC requests time out like any other requests using timeout or its default.
 		GrpcTimeoutHeaderMax: Notimeout,
 	}
+	out.GetRoute().RetryPolicy = &route.RetryPolicy{
+		RetryOn: "connect-failure,reset-before-request",
```

Comment:
About `connect-failure`, without setting inbound, it has already been covered with default outbound retry policy.

About `reset-before-request`, inbound retry on it can supplement cases where request has been sent to uypstream envoy, but envoy has not chance to send to server instance. 

So reset-before-request seems meaningful